src/api/data/helm/client/client.go

```diff
@@ -80,7 +56,13 @@ func (c *helmClient) initialize() (*helm.Client, error) {
 		tillerHost = fmt.Sprintf("localhost:%d", tunnel.Local)
 	}
 
-	return helm.NewClient(helm.Host(tillerHost)), nil
+	client := helm.NewClient(helm.Host(tillerHost))
+	// test connection
+	if _, err := client.GetVersion(); err != nil {
+		return nil, errors.New("Failed to connect to Tiller, are you sure it is installed?")
```

Comment:
error strings should not be capitalized. Read more about it in https://github.com/golang/go/wiki/CodeReviewComments#error-strings